x/incentive/types/params.go

```diff
@@ -177,3 +184,77 @@ func (rs Rewards) String() string {
 	}
 	return out
 }
+
+// Multiplier amount the claim rewards get increased by, along with how long the claim rewards are locked
+type Multiplier struct {
+	Name         MultiplierName `json:"name" yaml:"name"`
+	MonthsLockup int            `json:"months_lockup" yaml:"months_lockup"`
```

Comment:
`int` could introduce some indeterminsm over using `sdk.Int` or `int64`.
I think if someone's running a node on a super old 32 bit computer, golang uses int32 as the underlying type for int. Whereas on modern machines it uses int64.